spec/controllers/wikis_controller_spec.rb

```diff
@@ -0,0 +1,201 @@
+require 'rails_helper'
+include RandomData
+
+RSpec.describe WikisController, type: :controller do
```

Comment:
most of these tests I would not write in real life. These are good practice, though! We can go over tomorrow which tests I would and wouldn't if you are curious!